capa/features/extractors/ghidra/helpers.py

```diff
@@ -70,3 +75,156 @@ def get_function_symbols() -> Iterator[ghidra.program.database.function.Function
     """yield all non-external function symbols"""
 
     yield from currentProgram.getFunctionManager().getFunctionsNoStubs(True)  # type: ignore [name-defined] # noqa: F821
+
+
+def get_file_imports() -> Dict[int, Any]:
+    """get all import names & addrs"""
+
+    addrs = []
+    names = []
+
+    for f in currentProgram.getFunctionManager().getExternalFunctions():  # type: ignore [name-defined] # noqa: F821
+        for r in f.getSymbol().getReferences():
+            if r.getReferenceType().isData():
+                addr = r.getFromAddress().getOffset()  # gets pointer to fake external addr
+
+        fstr = f.toString().split("::")  # format: MODULE.dll::import / MODULE::Ordinal_*
+        if "Ordinal_" in fstr[1]:
+            fstr[1] = f"#{fstr[1].split('_')[1]}"
+
+        for name in capa.features.extractors.helpers.generate_symbols(fstr[0][:-4], fstr[1]):
+            addrs.append(addr)
+            names.append(name)
+
+    return dict(zip(addrs, names))
+
+
+def get_file_externs() -> Dict[int, Any]:
+    addrs = []
```

Comment:
do you have an example of this? if so, let's add a comment pointing to the example for future reference.